### PR TITLE
fix windows paths

### LINKED ISSUE #1: windows
doesn't quite work with font-awesome in windows yet. 

error looks like erorr, can't find : 

```
file:///C:/Usersmixmixprojectspatchbaynode_modules%0Font-awesome%0Fonts/fontawesome-webfont.woff2?v=4.7.0
```

### PR DESCRIPTION
**Pull request by mixmix**
fixes #1

### CODE REVIEW

**Comment by mixmix** [comment]
cc @ahdinosaur

File: index.js [hunk 1]
```diff
@@ -13,11 +13,17 @@ module.exports = requireStyle
 function requireStyle (name) {
   const path = styleResolve.sync(name)
   var css = readFileSync(path, 'utf8')
+  
+  const isWindows = /^win/.test(process.platform)
```
Comment (ahdinosaur) [inline index.js]: i would rather we use a module for this, like [`is-windows`](https://github.com/jonschlinkert/is-windows)

File: index.js [hunk 2]
```diff
@@ -13,11 +13,17 @@ module.exports = requireStyle
...
+  
   // resolve any relative paths to absolute paths
   css = css.replace(cssUrlRegex, (_, url) => {
     url = url
       .replace(quoteRegex, '')
       .replace(dotSlashRegex, match => join(dirname(path), match))
+    
+    if (isWindows) url = url.replace(/\\/g, '/')
```
Comment (ahdinosaur) [inline index.js]: would rather we use a module for this, like [`slash`](https://www.npmjs.com/package/slash).

**Comment by mixmix** [comment]
Okee, I'll get into those

**Comment by mixmix** [comment]
yep, put those in and smoke tested on windows. Works like a charm.
I meant to say it was hella exciting to copy that one fontAwesome css includer and then just have it in app.

**Comment by ahdinosaur** [comment]
sweet! only change now is to remove the `package-lock.json` you just committed (even if we wanted to commit _a_ lock file, _this_ lock file is incomplete)

**Comment by ahdinosaur** [comment]
@mixmix added you to GitHub repo and npm package, merge and publish when you're ready. :smile:

**Comment by mixmix** [comment]
Cool bra, thanks!

**Comment by mixmix** [comment]
merged and published as `1.0.1`
thanks for your help + support @ahdinosaur :heart:

**Pull request merged**